x-pack/test/functional/services/ml/data_visualizer_index_based.ts

```diff
@@ -34,7 +37,10 @@ export function MachineLearningDataVisualizerIndexBasedProvider({
 
     async assertFieldsPanelsExist(expectedPanelCount: number) {
       const allPanels = await testSubjects.findAll('~mlDataVisualizerFieldsPanel');
-      expect(allPanels).to.have.length(expectedPanelCount);
+      expect(allPanels).to.have.length(
+        expectedPanelCount,
+        `Expectedfield panels count to be '${expectedPanelCount}' (got '${allPanels.length}')`
```

Comment:
Typo - need space between `Expected` and `field`

Reply:
Fixed in 37f1b9f